yui/build/moodle-mod_hsuforum-article/moodle-mod_hsuforum-article-debug.js

```diff
@@ -1462,16 +1462,18 @@ M.mod_hsuforum.restoreEditor = function() {
             }
         }
 
-
-
+        // CTED-2779 | LMS Upgrade 2022 | Discussion Forum | Editor not saving styles
+        // Fix:
+        // The code below caused editor to be hidden when a toolbar submenu option was
+        // selected, example: click on a paragraph mode.
+        // --------------------------------------------------------------------------
         // Switch all editor links to hide mode.
-        M.mod_hsuforum.toggleAdvancedEditor(false, true);
-
-        // Put editor back in its correct place.
-        Y.one('#hiddenadvancededitorcont').show();
-        Y.one('#hiddenadvancededitorcont')._node.style.display='block';
-        editCont.appendChild(editor);
-        editCont.appendChild(Y.one('#hiddenadvancededitor'));
+        //M.mod_hsuforum.toggleAdvancedEditor(false, true);
+        // // Put editor back in its correct place.
+        // Y.one('#hiddenadvancededitorcont').show();
+        // Y.one('#hiddenadvancededitorcont')._node.style.display='block';
+        // editCont.appendChild(editor);
+        // editCont.appendChild(Y.one('#hiddenadvancededitor'));
```

Comment:
No need for this if the code is being removed completely. As long as branch naming conventions are followed, Jira will be able to associate the ticket to the branch and subsequent PRs, so we can see what was added/removed as part of the ticket.

Reply:
Hi Brendon, I was a bit wary of just deleting this code. But the editor does work without it, so, i will completely remove this block and we can easily track changes and bring it back if something else gets broken.